src/shared/components/home/home.tsx

```diff
@@ -137,6 +139,7 @@ export class Home extends Component<any, HomeState> {
     dataType: getDataTypeFromProps(this.props),
     sort: getSortTypeFromProps(this.props),
     page: getPageFromProps(this.props),
+    tagline: ""
```

Comment:
`tagline: None`

Reply:
Out of curiosity, wouldn't just allowing empty string be simpler? no need to match in the html on None/Some to handle displaying differently?

Reply:
javascript null checking is a weird nightmare, people even have to write helper functions for empty / blank string checks. Much safer to use the monad options wherever possible for missing / null values.